pkg/controller/collector/set.go

```diff
@@ -12,6 +14,7 @@ type SetConfig struct {
 	Logger               micrologger.Logger
 	K8sClient            k8sclient.Interface
 	NewRuntimeObjectFunc func() runtime.Object
+	Selector             selector.Selector
```

Comment:
This is basically a breaking change, or do I misunderstand?

Leaving this empty will cause no metrics to be collected AFAICT

Reply:
It's not. If you don't pass a selector to the controller it defaults to a selector that returns everything ([link](https://github.com/giantswarm/operatorkit/blob/cfe34c7b88e5d94355e15778f9e0545b93344144/pkg/controller/controller.go#L166-L168)) which will get propagated to the [controller set](https://github.com/giantswarm/operatorkit/blob/cfe34c7b88e5d94355e15778f9e0545b93344144/pkg/controller/controller.go#L185) too.